### DISCUSSION TITLE
Search results order

### DISCUSSION THREAD

**Discussion by ghost** [libre-tube/LibreTube#3679] [Q&A] [answered]
It would be nice to be able to sort the search results according to some chriteria as well as in the youtube app. Like for example, descendent by date.

**Comment by FireMasterK** [comment, marked as answer]
We need https://github.com/TeamNewPipe/NewPipeExtractor/pull/904 to be merged first

**Reply by ghost** [reply]
Thank you very much for the info. Let's see if they merge soon.
